test/e2e-test/profile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.12.1-alpine3.19
+
+ARG USER=ccnp
+ARG UID=1000
+ARG GID=1000
+ARG GROUP=ccnp
+
+WORKDIR /run/confidential-cloud-native-primitives
+COPY  ./ ./
+RUN addgroup -S -g $GID $GROUP && adduser -S -u $UID -D -G $GROUP $USER
+RUN chown -R  $USER:$GROUP  ./
+
+RUN apk update && apk add bash vim
+<<<<<<< HEAD
```

Comment:
Seems this is an unresolved conflict.

Reply:
Thanks so much!  Just resolved it.  https://github.com/cc-api/confidential-cloud-native-primitives/blob/main/test/e2e-test/profile/Dockerfile